int64 collision_detected

